### ISSUE TITLE
GuideGuide with Photoshop CC 2015 - Extension not showing up

### ISSUE THREAD

**Issue by DecoMeOut** [guideguide/support#186]
I experienced the glory of GuideGuide for only one project before I updated Adobe CC to 2015 and now I can't for the life of me get the extension to show up in Photoshop! I've deleted and reinstalled it about 4 times, following the instructions carefully each time and restarting my computer each time. My Extension Manager is up to date. I dragged the nested folder next to the Read Me file as instructed. I read all of the threads on this forum that I could find - no luck there. Every time I reopen Photoshop and check Window>Extensions I have a glimmer of hope. It's such a fantastic extension I must have it back! See a screenshot of my folder structure attached.
<img width="1663" alt="screen shot 2015-09-18 at 12 48 57 am" src="https://cloud.githubusercontent.com/assets/14343960/9954658/a284903c-5d9f-11e5-8a79-4ad434ce4c27.png">

**Comment by cameronmcefee** [comment]
Hi @DecoMeOut,

I'm sorry to hear of your trouble. From your screenshot, it definitely seems like you've got it installed correctly. In fact, your message basically answers all of my usual questions.

The bad news is, I have no idea what's up. However, I can provide you with some information that you might be able to use to troubleshoot things yourself:

- First, the fact that you don't see GuideGuide in the Window > Extensions menu tells me that it is either not in the correct folder, or Photoshop cannot read the manifest file (in the CSXS folder). Your screenshot tells me that you installed it where I would expect you should install it, which leads me to believe Photoshop is not looking for extensions in its normal place. We should confirm whether that folder is used for extensions.

  I see you have other extensions in that folder. Can you use those extensions in Photoshop? If so, close Photoshop, drag those folders to your desktop, then relaunch Photoshop. If this is really Photoshop's extensions folder, those extensions should no longer work because you moved them. Once you've performed this test, move the folders back so they'll work again.

  If removing the folders broke those extensions, that tells us the folder is right, and the problem is specific to GuideGuide. If removing those folders didn't break things, then that tells us the installation folder is somewhere else. Let me know what you find.

- I noticed Photoshop has a setting to allow extensions. I would be unlikely that you'd have turned it off, but just to be sure, check out Preferences > Plug-ins and make sure "Load Extension Panels" is checked.

**Comment by DecoMeOut** [comment]
Interesting! Load Extensions Panels is indeed checked. 

My Window>Extensions panel only shows one Extension (Adobe Colour Themes) even though there are 4 extensions in my extensions folder. Moving all of those folders to the desktop did not change this fact. So I'm thinking you're right that my installation folder is somewhere else. Any idea how to locate it?

**Comment by cameronmcefee** [comment]
@DecoMeOut it just occurred to me, not all of those will appear in the extensions menu. Can you perform the same test, but locate Device Preview in Window > Device Preview first. That's what should break when you move com.adobe.preview.

**Comment by DecoMeOut** [comment]
Device Preview became unavailable when I moved the folder to desktop. Does this complicate things?

**Comment by cameronmcefee** [comment]
It complicates things in that you've confirmed that the install folder is correct.

I don't supposed you modified any of the files while installing? Or copied them in a way that might have modified them?

**Comment by DecoMeOut** [comment]
I dragged and dropped the entire folder from one finder window to another.

**Comment by cameronmcefee** [comment]
Is there anything in your equivalent user Library folder?

**Comment by DecoMeOut** [comment]
Sorry I don't know what you mean.

**Comment by cameronmcefee** [comment]
Ah, my apologies. OS X has both *system* and *user* Library folders which mirror one another in terms of contents, the latter being specific to you, the former specific to every user. Some times there are conflicts when GuideGuide exists in both the users and system folders. You open this folder quickly by running this in the terminal:

```
open ~/Library/Application\ Support/Adobe/CEP/extensions/
```

If you see any GuideGuide folders in the folder that opens up, you can delete them.

Beyond this, I don't really have any suggestions other than trying to reinstall Photoshop itself.

**Comment by DecoMeOut** [comment]
Success! Thank you for all your help! Your advice led me to figure out what the problem was :) 
Here's what happened:
I finally noticed that my GuideGuide folder in extensions had a little black arrow in the corner - indicating that it was just linking from another location. So I deleted the linked folder from extensions and emptied my trash. Then I right-clicked "Copy GuideGuide" on the original GuideGuide folder and 'Paste item' into my extensions folder (instead of dragging and dropping). Finally I deleted all of the GuideGuide files and folders that were not in my extensions folder. Emptied trash. When I restarted Photoshop there it was!
